src/commands/oas2ts.js

```diff
@@ -0,0 +1,86 @@
+import { Command } from 'commander'
+import fs from 'fs-extra'
+import { runCommand as runJson2TsCommand } from './json2ts.js'
+import { runCommand as runOas2JsonCommand } from './oas2json.js'
+
+const TEMP_FOLDER = 'temp-json-schemas'
+
+const cleanUpTempFolder = () => {
+  fs.removeSync(TEMP_FOLDER)
```

Comment:
can we use the OS's temp folder instead of a folder in the same location where this is run? so we don't even have to clean it up after

Reply:
I updated the code to use `os.tmpdir()` as `TEMP_FOLDER` but it seems it's still a good practice to clean it up manually.